.eleventy.js

```diff
@@ -1,7 +1,8 @@
 const buildTOC = require('./src/BuildTOC')
+const parseOptions = require('./src/ParseOptions')
 
-module.exports = function(eleventyConfig, pluginNamespace) {
-  eleventyConfig.namespace(pluginNamespace, () => {
-    eleventyConfig.addFilter('toc', (content, opts) => buildTOC(content, opts))
+module.exports = (eleventyConfig, globalOpts = {}) => {
```

Comment:
Turns out the global options are in the 2nd param (which is `pluginNamespace` in most plugins. We can use that and still run the `.namespace` function:

```js
module.exports = (eleventyConfig, globalOpts) => {
  globalOpts = globalOpts || {}
  eleventyConfig.namespace(globalOpts, () => {
    eleventyConfig.addFilter('toc', (content, localOpts) => {
      return buildTOC(content, parseOptions(localOpts, globalOpts))
    })
  })
}
```

Reply:
@jdsteinbach got it, didn't consider it but I just updated the code. Hopefully we're good to :shipit:! 😁